index/segment/fst/roaring_postings_codec.go

```diff
@@ -0,0 +1,63 @@
+// +build disabled
+// TODO(prateek): measure perf of this v pilosa
```

Comment:
rudimentary tests show its 2x worse. should drop it.

Reply:
glad to hear the pilosa library is quite good 👍, definitely interested to hear about what you observed at some point 📉

Reply:
For sure. So using the tests as a poor man's benchmark:

```
# pilosa 
❯ time go test ./index/segment/tests -count=10
ok  	github.com/m3db/m3ninx/index/segment/tests 2.192s
/Users/prungta/code/go1.10.2/bin/go test ./index/segment/tests -count=10  3.81s user 0.47s system 131% cpu 3.266 total

# roaring binary 
❯ time go test ./index/segment/tests -count=10
ok  	github.com/m3db/m3ninx/index/segment/tests 1.966s
/Users/prungta/code/go1.10.2/bin/go test ./index/segment/tests -count=10  3.76s user 0.46s system 137% cpu 3.073 total


# roaring msgpack 
❯ time go test ./index/segment/tests -count=10
ok  	github.com/m3db/m3ninx/index/segment/test 8.219s
/Users/prungta/code/go1.10.2/bin/go test ./index/segment/tests -count=10  17.38s user 0.78s system 197% cpu 9.211 total
```

conclusions:
- upstream roaring msgpack is a pos
- binary perf of upstream roaring v pilosa is comparable. 

This is despite the fact that in the case of the pilosa tests, we're only using pilosa for serialization; and always convert from []byte -> pilosa -> upstream roaring (and vice versa during writes) to satisfy existing interfaces. So once we switch our postings implementation to pilosa, it'd be able to avoid a bunch of things it's having to do now. 

Also checked mem usage, binary roaring v pilosa were comparable - 94MB v 105MB alloc space (again, despite the extra work our pilosa impl had to do for conversions)
<img width="1046" alt="roaring-binary-mem" src="https://user-images.githubusercontent.com/199982/39968454-6d3f976e-56bc-11e8-9725-7c43fb479b88.png">
<img width="979" alt="pilosa-mem" src="https://user-images.githubusercontent.com/199982/39968455-6d555d7e-56bc-11e8-8e9b-6f0339059354.png">